documentation/docs/concepts/decentralized_identifiers/update.mdx

```diff
@@ -46,363 +50,362 @@ methods allow them to be used by more than one verification relationship.
 Upon creating a verification method using the identity framework, specifying the `MethodScope` option will result in an embedded verification method. Leaving that option unset will create the verification method as
```

Comment:
The "embedded or referenced" URL above in the sentence before this (I can't highlight it) is a broken link, the reference changed.

New link:
https://www.w3.org/TR/did-core/#referring-to-verification-methods